src/app/SIPUserAgents/SIPRegistrationUserAgent.cs

```diff
@@ -202,13 +202,6 @@ public void Stop()
 
                     m_exit = true;
                     m_waitForRegistrationMRE.Set();
-
-                    if (m_isRegistered)
-                    {
-                        m_attempts = 0;
-                        m_expiry = 0;
-                        ThreadPool.QueueUserWorkItem(delegate { SendInitialRegister(); });
-                    }
                 }

```

Comment:
This block is needed. It's purpose is to send a REGISTER request with 0 expiry when the agent is shut down. This de-registers the agent with the server. If this request does not get sent it could take up to an hour for the server to expire the registration.

Reply:
Fixed.
My bad. I forgot about this.